plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -107,3 +120,28 @@ export const formatPRsByReviewDecision = (
     { title: COLUMNS.APPROVED, content: reviewDecisions.APPROVED },
   ];
 };
+
+export const shouldDisplayCard = (
+  repository: Repository,
+  author: Author,
+  teamRepositories: string[],
+  teamMembers: string[],
+  infoCardFormat: PRCardFormating[],
+  isDraft: boolean,
```

Comment:
should isDraft ONLY show drafts, or show everything including drafts? maybe this could be a tri-state thing instead of a bool :) `drafts: 'ignore' | 'include' | 'only'` (just a thought, feel free to ignore)

Reply:
I think that'd be pretty neat to have that, I wish we were able to have both drafts and non-draft PR on the same view in the current dashboard.
But I think that falls outside the scope of this PR (the current implementation already relies on a boolean), though I'm interested to work on that on a separate PR in the future 🤔